.github/workflows/code_reviewer.yml

```diff
@@ -0,0 +1,21 @@
+name: AI PR Reviewer
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+permissions: write-all
+jobs:
+  tc-ai-pr-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: TC AI PR Reviewer
+        uses: topcoder-platform/tc-ai-pr-reviewer@master
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # The GITHUB_TOKEN is there by default so you just need to keep it like it is and not necessarily need to add it as secret as it will throw an error. [More Details](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret)
```

Comment:
The comment about the `GITHUB_TOKEN` is informative but might be better placed in documentation or a README file rather than in the workflow file itself. Consider removing it from here to keep the workflow file clean.